app/controllers/posts_controller.rb

```diff
@@ -5,10 +5,23 @@ def index
     @comments = Comment.where(author_id: params[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
   def show
     @post = Post.find(params[:id])
     @comments = Comment.all
     @user = User.find(params[:user_id])
     @all_users = User.all
   end
+
```

Comment:
### [OPTIONAL]
With strong parameters, Action Controller parameters are forbidden to be used in Active Model mass assignments until they have been permitted. This means that you'll have to make a conscious decision about which attributes to permit for mass update. This is a better security practice to help prevent accidentally allowing users to update sensitive model attributes.

Lesson link about the strong parameters: https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.
```ruby
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```